types/i18next-express-middleware/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: http://i18next.com/
 // Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```

Comment:
Default generics are used, so yeah, @GiedriusGrabauskas is right. This line is needed.